Forms/Tab.php

```diff
@@ -115,7 +122,10 @@ public function getTabSet() {
 	}
 
 	public function extraClass() {
-		return implode(' ', (array)$this->extraClasses);
+		$classes = (array)$this->extraClasses;
+		$classes[] = 'tab-pane';
```

Comment:
Yes, this is a real shame... :( The alternative way was to use entwine and a lot of "onmatch" and "onunmatch" checking, which I thought was the worse of two evils.
I went with this implementation and got away with only one "onmatch" checking required:
https://github.com/silverstripe/silverstripe-framework/pull/5964/files#diff-8152de84382846fefee15697ffa38b6dR79